app/views/shared/_tagging.html.erb

```diff
@@ -0,0 +1,104 @@
+
+        <div class="form-group" id="edition_browse_pages_input">
+          <%= f.label :mainstream_browse_pages, "Mainstream browse pages", class: 'control-label' %>
+          <%= f.select :mainstream_browse_pages,
+                       Tagging::Linkables.mainstream_browse_pages,
+                       {},
+                       { multiple: true,
+                         class: 'js-mainstreams select2',
+                         data: { placeholder: 'Choose mainstream browse pages…' } } %>
+
+          <p class='help-block'>
+            Mainstream browse pages live under <a href="https://www.gov.uk/browse">/browse</a>.
+            Examples: <a href="https://www.gov.uk/browse/working">Working</a> or
+            <a href="https://www.gov.uk/browse/business">Business</a>.
+            They are used to organise mainstream content. Formerly called
+            <i>sections and subsections</i>.
+          </p>
+        </div>
+
+        <div class="form-group" id="edition_parent_input">
+          <%= f.label :parent, "Breadcrumb", class: 'control-label' %>
+          <%= f.select :parent,
+                       Tagging::Linkables.mainstream_browse_pages,
+                       { include_blank: true },
+                       { multiple: false,
+                         class: 'js-primary-mainstream select2',
+                         data: { placeholder: 'Choose a primary mainstream browse page…' } } %>
+
+         <br />
+         <div class="alert mainstream alert-warning hidden">
+           The breadcrumb does not match the browse pages.
+         </div>
+
+
+          <p class='help-block'>
+            If the page has a breadcrumb, it will be this browse page. For example
+            <a href="https://www.gov.uk/benefit-cap-calculator">the breadcrumb on this page</a>.
+            You can only choose one.
+          </p>
+        </div>
+
+        <div class="form-group" id="edition_topics_input">
+          <%= f.label :topics, "Topics", class: 'control-label' %>
+          <%= f.select :topics, Tagging::Linkables.topics,
+                       {},
+                       { multiple: true,
+                         class: 'select2',
+                         data: { placeholder: 'Choose Topics…' } } %>
+
+          <p class='help-block'>
+            Topic pages live under <a href="https://www.gov.uk/topic">/topic</a>.
+            They contain collections of pages on GOV.UK. Example:
+            <a href="https://www.gov.uk/topic/business-tax">Business Tax</a> or
+            <a href="https://www.gov.uk/topic/animal-welfare/pets">Animal Welfare / Pets</a>.
+            Tagging a page with a topic will send out an
+            <a href="https://www.gov.uk/topic/business-tax/vat/email-signup">email alert to subscribers</a>.
+            Formerly called <i>specialist sectors</i>.
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <hr/>
+    <%= f.submit 'Update tags' %>
+<% end %>
+
+
+<script type="text/javascript">
```

Comment:
Content Tagger and Publisher are meant to have the same logic in the tagging form. This form has been heavily "inspired" on the way content tagger has been done. It's easier to come to this one file and know anything that is directly related, I'm not sure when someone will make this a shared resource between projects but I feel this should be directly in this file for the sake of the next person that picks this up.
I believe what you suggesting is creating another file and but bear in mind that application.js requires all the js files in the directory, that basically is loading files that are not needed to be loaded all the time.